app/store/services/helpers/gps.js

```diff
@@ -25,20 +25,19 @@ const isInArea = (
   Math.abs(position.longitude - point.longitude) < Δλ &&
   Math.abs(position.latitude - point.latitude) < Δφ;
 
-const getEventDistance = (points = []) => {
+const computePathDistance = (points = []) =>
   points.reduce((distance, currentPoint, index) => {
     const nextPoint = points[index + 1];
     if (nextPoint) {
-      return distance + getDistance(currentPoint, nextPoint);
+      return distance + computeDistance(currentPoint, nextPoint);
     }
     return distance;
   }, 0);
-};
 
-const getNearestPoint = (points = [], currentPosition = {}) => {
+const findClosestPoint = (points = [], currentPosition = {}) => {
```

Comment:
findClosestNode?

Reply:
but i think you need
findClosestEdge (note the math are not the same)

Reply:
We need to pair programming about that! 
Not 100% agree about that!